test/schema.json

```diff
@@ -0,0 +1,475 @@
+          "type": "boolean",
+          "description": "Normally not set. A flag to use during development to only run one or more specific tests."
+        }
+      }
+    },
+    "description": {
+      "type": "string",
+      "description": "Information about the test."
+    },
+    "testType": {
+      "description": "The category of test.",
+      "enum": [
+        "syntax",
+        "function"
+      ]
+    },
+    "locale": {
+      "description": "The locale to use for formatting.",
+      "type": "string"
+    },
+    "src": {
+      "description": "The MF2 syntax source.",
+      "type": "string"
+    },
+    "params": {
+      "description": "Parameters to pass in to the formatter for resolving external variables.",
+      "type": "array",
+      "items": {
+        "$ref": "#/$defs/var"
+      }
+    },
+    "var": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "type",
+        "value"
+      ],
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "type": {
+          "enum": [
+            "boolean",
+            "datetime",
+            "null",
+            "number",
+            "string"
+          ]
+        },
+        "value": {}
+      },
+      "oneOf": [
+        {
+          "properties": {
+            "type": {
+              "const": "boolean"
+            },
+            "value": {
+              "type": "boolean"
+            }
+          }
+        },
+        {
+          "properties": {
+            "type": {
+              "const": "datetime"
+            },
+            "value": {
+              "type": "string"
+            }
+          }
+        },
+        {
+          "properties": {
+            "type": {
+              "const": "null"
+            },
+            "value": {
+              "type": "null"
+            }
+          }
+        },
+        {
+          "properties": {
+            "type": {
+              "const": "number"
+            },
+            "value": {
+              "type": "number"
+            }
+          }
+        },
+        {
+          "properties": {
+            "type": {
+              "const": "string"
+            },
+            "value": {
+              "type": "string"
+            }
+          }
+        }
+      ]
+    },
+    "exp": {
+      "description": "The expected result of formatting the message to a string.",
+      "type": "string"
+    },
+    "expCleanSrc": {
+      "description": "The expected normalized form of `src`, for testing stringifiers.",
+      "type": "string"
+    },
+    "expParts": {
+      "description": "The expected result of formatting the message to parts.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "type"
+        ],
+        "properties": {
+          "type": {
+            "enum": [
+              "markup",
+              "literal",
+              "fallback"
+            ]
+          },
```

Comment:
The parts can have much more varied shapes. See [here](https://github.com/messageformat/messageformat/blob/main/packages/mf2-messageformat/src/formatted-parts.ts) for a TypeScript definition.

Reply:
Thanks - I'll update the schema accordingly